dm/tests/upstream_switch/case.sh

```diff
@@ -2,6 +2,7 @@
 
 set -eu
 
+export DM_MASTER_EXTRA_ARG=""
```

Comment:
Will it overwrite the DM_MASTER_EXTRA_ARG set by GitHub secret?

Reply:
just same with other test https://github.com/pingcap/tiflow/blob/68c22f32b2090e83e5ae64ae3b1f25f1a9575ff8/dm/tests/run.sh#L6